lib/tutorials/en_US/cookies.md

```diff
@@ -1,59 +1,98 @@
+
+## <a name="server" /> Configuring the server
+
+hapi has several configurable options when dealing with cookies. The defaults are probably good for most cases, but can be changed when needed. To prepare a cookie, you first need to name it and configure a list of options.  You do this by calling `server.state(name, [options])`.
+
+### <a name="server.state" /> server.state()
+
+To use a cookie, you first need to configure it by calling [`server.state(name, [options])`](/api#-serverstatename-options) where `name` is the name of the cookie, and `options` is an object used to configure the cookie:
 
 ```javascript
 server.state('data', {
     ttl: null,
     isSecure: true,
     isHttpOnly: true,
     encoding: 'base64json',
-    clearInvalid: false, // remove invalid cookies
-    strictHeader: true // don't allow violations of RFC 6265
+    clearInvalid: true,
+    strictHeader: true
 });
 ```
+In the example above, we first name the cookie `data`. We then configure the options to customize the cookie.
+
+The first option is `ttl`. This represents the cookies time to live in milliseconds. The default setting is `null`, which will delete the cookie once the browser is closed.
+
+Next we have two flags, `isSecure` and `isHttpOnly`. For more info on these flags, see ([RFC 6265](http://tools.ietf.org/html/rfc6265), specifically sections [4.1.2.5](http://tools.ietf.org/html/rfc6265#section-4.1.2.5) and [4.1.2.6](http://tools.ietf.org/html/rfc6265#section-4.1.2.6).
+
+We then tell hapi that the value is a base64 encoded JSON string.
+
+By setting `clearInvalid` to `true`, it instructs the client to remove invalid cookies.
+
+Last, we set `strictHeader` to `true` so that there are no violations of [RFC 6265](https://tools.ietf.org/html/rfc6265).
 
-This configuration will make it so the cookie named `data` has a session time-life (will be deleted by the browser when closed), is flagged both secure and HTTP only (see [RFC 6265](http://tools.ietf.org/html/rfc6265), specifically sections [4.1.2.5](http://tools.ietf.org/html/rfc6265#section-4.1.2.5) and [4.1.2.6](http://tools.ietf.org/html/rfc6265#section-4.1.2.6) for more details about these flags), and tells hapi that the value is a base64 encoded JSON string. Full documentation for the `server.state()` options can be found in [the API reference](/api#serverstatename-options).
+### <a name="options.state" /> route.options.state()
 
-In addition to this, you can further configure cookie behaviour at a route-level by specifying two properties at the route's `options.state` object:
+In addition to this, you can further configure cookie behavior at a route-level by specifying two properties at the route's `options.state` object:
```

Comment:
I’d like to add a note that the route-level cookie configurations are an addition to the `server.state` config.

Is it true what I’m saying? If yes, what do you think?

Reply:
Yeah, I can see how this could get confusing